packages/mrgn-utils/src/actions/flashloans/helpers.ts

```
@@ -243,44 +231,60 @@ export async function calculateMaxRepayableCollateralLegacy(
 }
 
 export async function calculateMaxRepayableCollateral(
-  borrowBank: ExtendedBankInfo, // USDC
-  depositBank: ExtendedBankInfo, // JITOSOL
+  borrowBank: ExtendedBankInfo,
+  depositBank: ExtendedBankInfo,
   slippageBps: number,
   slippageMode: "DYNAMIC" | "FIXED"
 ): Promise<{ amount: number; maxOverflowHit: boolean }> {
-  if (!depositBank.isActive || !borrowBank.isActive) return { amount: 0, maxOverflowHit: false };
+  // if the bank is not active, a bug is occurring.
+  if (!depositBank.isActive || !borrowBank.isActive) {
+    console.error(
+      "An internal configuration issue has occurred: Bank is not active. Please create a support ticket for assistance."
+    );
+    return { amount: 0, maxOverflowHit: false };
+  }
+
+  // if slippage mode is fixed and no slippage is provided, return 0
+  if (slippageMode === "FIXED" && slippageBps === 0) {
+    console.error(
```

We have a couple of console.errors in here, is this handled or does this fail silently? Should we display an error or warning toasts maybe? Or style the console.error a bit differently maybe? This could be missed easilly by the user